packages/lib-classifier/src/components/Classifier/components/SubjectViewer/components/FlipbookViewer/components/FlipbookControls.js

```diff
@@ -1,29 +1,53 @@
-import { Button, Box, Grid } from 'grommet'
-import { FormNext, FormPrevious } from 'grommet-icons'
+import { Button, Box, Grid, Select, ThemeContext } from 'grommet'
+import {
+  CirclePlay,
+  Pause,
+  FormNext,
+  FormPrevious,
+  FormDown
+} from 'grommet-icons'
 import PropTypes from 'prop-types'
-import React from 'react'
+import React, { useEffect, useRef, useState } from 'react'
 import styled, { withTheme } from 'styled-components'
 import { useTranslation } from 'react-i18next'
 
+import controlsTheme from './theme'
 import locationValidator from '../../../helpers/locationValidator'
 
-const StyledButton = styled(Button)`
+const DirectionButton = styled(Button)`
   & > div {
```

Comment:
HTML buttons can’t contain `div` tags so this component is generating invalid HTML somewhere. React will warn about invalid DOM nesting errors when this happens.

Reply:
The invalid HTML is a result of Grommet's Button handling an `icon` and `label` prop. Would you recommend creating my own button component instead?

Reply:
It's not a big deal. HTML parsers can handle invalid button content, probably by ignoring the invalid block layout. Different browsers might handle a `<div>` inside a button differently, so the layout isn't guaranteed to be the same cross-browser.

The bug is caused by this Grommet `Box`. One way to avoid it is to explicitly pass children to the button, rather than `icon` and `label`. We can also open an issue with Grommet, and a PR to fix this.
https://github.com/grommet/grommet/blob/f9c351e1bb7d55a18f94cc60c017fac50d26f7e3/src/js/components/Button/Button.js#L339-L357